include/nfc/t4t/isodep.h

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef NFC_T4T_ISODEP_H_
+#define NFC_T4T_ISODEP_H_
+
+/**
+ * @file
+ * @defgroup nfc_t4t_isodep NFC Type 4 Tag ISO-DEP
+ * @{
+ * @brief NFC Type 4 Tag ISO-DEP API.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup isodep_error ISO-DEP Protocol error
+ * @{
+ */
+/** Correct frame was received with invalid content. */
+#define NFC_T4T_ISODEP_SYNTAX_ERROR 1
+
+/** Correct frame was received when it is not expected. */
+#define NFC_T4T_ISODEP_SEMATIC_ERROR 2
+
+/** Unrecoverable Transmission error. */
```

Comment:
why transmission and in second comment timeout start from capital letter?

Reply:
can be left as is but it is inconsistency with other comments

Reply:
Changed